index.html

```diff
@@ -22,16 +22,17 @@
   <header>
     <div class="row">
       <div class="column-full">
-        <div class="container">
+        <div class="container nav-bar">
           <h3>Code Journal</h3>
+          <h3 class="entries-anchor">Entries</h3>
```

Comment:
This should be an actual anchor element, so it should be using the `a` element. You can just set the `href` to a `#`.